### PR TITLE
docs: prevent to delete other clusters data on cluster deletion

### PR DESCRIPTION
**Pull request by satoru-takeuchi**
**Description of your changes:**

Current cluster teardown document assumes that there is only one Rook/Ceph cluster. If not, the instruction written in this document might delete the data of other clusters.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [x] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [x] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [x] Unit tests have been added, if necessary.
- [x] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by satoru-takeuchi** [comment]
@galexrt I hesitate to use `ceph device ls` here. This command shows kernel device names which might be changed after node restart. The worst scenario is as follows.

1. A user gets node:dev pairs from`ceph device ls`.
2. An node is restarted during teardown process and an osd's device name on this node is changed from sdb to sdc.
3. This user wipes sdb, it's an unrelated device, by mistske.

**Review by travisn — CHANGES_REQUESTED** [review]
Just a few small suggestions on the wording

File: Documentation/Storage-Configuration/ceph-teardown.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ If the `cleanupPolicy` was applied and the cleanup jobs have completed on all th
 !!! attention
     The final cleanup step requires deleting files on each host in the cluster. All files under the `dataDirHostPath` property specified in the cluster CRD will need to be deleted. Otherwise, inconsistent state will remain when a new cluster is started.
 
-Connect to each machine and delete `/var/lib/rook`, or the path specified by the `dataDirHostPath`.
+Connect to each machine and delete `/var/lib/rook/rook-ceph`, or the path specified by the `rook-ceph` directory under `dataDirHostPath`.
```
[the remark this answers is not in the document]
Comment (travisn) [reply Documentation/Storage-Configuration/ceph-teardown.md:87]: one more update, thanks ^

[the remark this answers is not in the document]
Comment (satoru-takeuchi) [reply Documentation/Storage-Configuration/ceph-teardown.md]: Oops, sorry. I forgot to stage this change before commit.

**Review by travisn — APPROVED** [review]

**Review by galexrt — APPROVED** [review]

**Comment by galexrt** [comment]
@satoru-takeuchi good point regarding the risks of `ceph device ls` thanks!

**Pull request merged**